src/lib/device-state.ts

```diff
@@ -19,22 +28,23 @@ export const setMinPollInterval = (config: AnyObject) => {
 
 export const getReleaseForDevice = (
 	api: PinejsClient,
-	device: AnyObject,
+	device: Device,
```

Comment:
Changing the types of the method args in this file, will cause the resin-api typings to fail since they will still be passing `AnyObject` is some cases. I'm not sure how you prefer to handle this
* Require the actual type and the resin-api will have to update its typings. If we go this way, we should probably change this to be a minor change type, (which since we are on < v1.0.0, it is typically allowed to contain some breaking changes).
* Use `AnyObject | Device` so that pre-existing consumer projects still work.
* <your reviewer suggestion here>